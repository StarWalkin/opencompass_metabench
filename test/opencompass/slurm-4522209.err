  0%|          | 0/1 [00:00<?, ?it/s]                                       0%|          | 0/1 [00:00<?, ?it/s]slurmstepd: error: *** JOB 4522209 ON gpub079 CANCELLED AT 2024-08-18T00:45:52 ***
