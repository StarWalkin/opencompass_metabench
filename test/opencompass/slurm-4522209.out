/scratch/bcwg/Metabench/anaconda/envs/opencompass/bin/python3
08/18 00:28:43 - OpenCompass - INFO - Loading gsm8k_gen: configs/datasets/gsm8k/gsm8k_gen.py
08/18 00:28:43 - OpenCompass - INFO - Loading hf_gemma2_9b_it: configs/models/gemma/hf_gemma2_9b_it.py
08/18 00:28:43 - OpenCompass - INFO - Loading example: configs/summarizers/example.py
08/18 00:28:43 - OpenCompass - INFO - Current exp folder: outputs/default/20240818_002843
08/18 00:28:43 - OpenCompass - WARNING - SlurmRunner is not used, so the partition argument is ignored.
08/18 00:28:43 - OpenCompass - INFO - Partitioned into 1 tasks.
